src/vectorsynchronizer.js

```diff
@@ -81,6 +81,22 @@ olcs.VectorSynchronizer.prototype.createSingleCounterpart = function(olLayer) {
     csPrimitives.show = olLayer.getVisible();
   });
 
+  var onOpacityChange = function() {
```

Comment:
This code is unnecessary, on layer change, all the layer features are destroyed and created anew.
A finer mechanism in ol3-cesium would require the OL3 change event to be more detailed.